modules/nw-ingress-gateway-api-manage-succession.adoc

```diff
@@ -0,0 +1,62 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-gateway-api-manage-succession_{context}"]
+= Preparing for Gateway API management succession by the Ingress Operator
+
+Starting in {product-title} 4.19, the Ingress Operator manages the lifecycle of any Gateway API custom resource definitions (CRDs). Updating from a version before 4.19 of {product-title} where this management was not present requires you to replace or remove any Gateway API CRDs that already exist in the cluster so that they conform to the specific {product-title} specification required by the Ingress Operator. {product-title} version 4.19 requires Gateway API Standard version 1.2.0 CRDs.
```

Comment:
As noted in https://github.com/openshift/openshift-docs/pull/93327#pullrequestreview-2868138981, we should be more explicit that "the Ingress Operator manages the lifecycle of any Gateway API custom resource definitions" includes that the platform will deny anyone (including the cluster-admin) access to create, update, or delete any CRDs with the API groups that belong to Gateway API.